packages/fast-jss-manager-react/src/jss-manager.spec.tsx

```diff
@@ -60,201 +72,127 @@ const staticAndDynamicStyles: ComponentStyles<any, any> = {
-        color: "red",
-    };
+    test("should render the managedComponent", () => {
+        const rendered: any = mount(<NoPropsManager />);
 
-    function functionStyleSheet(config: TestDesignSystem): any {
-        return {
-            class: {
-                color: config.color,
-            },
-        };
-    }
-
-    test("should not throw when no stylesheet is provided", (): void => {
-        expect(
-            (): void => {
-                mount(<JSSManager render={renderChild} />);
-            }
-        ).not.toThrow();
+        expect(rendered.find("div")).toHaveLength(1);
+        expect(rendered.find("SimpleComponent")).toHaveLength(1);
     });
 
-    test("should not throw when no stylesheet is provided and design system is changed", (): void => {
-        const rendered: ReactWrapper = mount(
-            <JSSManager designSystem={testDesignSystem} render={renderChild} />
-        );
+    test("should pass props to managed component", (): void => {
+        const rendered: any = mount(<NoPropsManager id="id" />);
 
-        expect(
-            (): void => {
-                rendered.setProps({ designSystem: { color: "blue" } });
-            }
-        ).not.toThrow();
+        expect(rendered.find("SimpleComponent").prop("id")).toBe("id");
     });
 
-    test("should compile a stylesheet when mounting", (): void => {
-        const objectStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                render={renderChild}
-                designSystem={testDesignSystem}
-                styles={stylesheet}
-            />
-        );
-
-        const functionStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                render={renderChild}
-                styles={functionStyleSheet}
-                designSystem={testDesignSystem}
-            />
+    test("should not pass the jssStyleSheet prop to managed component", (): void => {
+        const rendered: any = mount(
+            <NoPropsManager jssStyleSheet={{ foo: { color: "red" } }} />
         );
 
-        expect(objectStylesheetComponent.state("styleSheet")).toBeInstanceOf(StyleSheet);
-        expect(objectStylesheetComponent.state("styleSheet").attached).toBe(true);
-
-        expect(functionStylesheetComponent.state("styleSheet")).toBeInstanceOf(
-            StyleSheet
-        );
-        expect(functionStylesheetComponent.state("styleSheet").attached).toBe(true);
+        expect(rendered.find("SimpleComponent").prop("jssStyleSheet")).toBeUndefined();
     });
 
-    test("should update an object stylesheet when the design-system changes", (): void => {
-        const objectStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                designSystem={{ color: "blue" }}
-                styles={stylesheet}
-                render={renderChild}
-            />
-        );
+    test("should not pass the managedClasses prop through to managed component", (): void => {
```

Comment:
likewise here "should not pass the managedClasses prop through to a managed component without styles"

Reply:
oh - not this is correct. It means what its saying.